app/src/main/java/mozilla/lockbox/support/ClipboardSupport.kt

```diff
@@ -0,0 +1,36 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package mozilla.lockbox.support
+
+import android.content.ClipData
+import android.content.ClipboardManager
+import android.content.Context
+
+typealias ClipboardSupportFactory = (Context) -> ClipboardSupport
+
+open class ClipboardSupport(
+    private val clipboard: ClipboardManager
+) {
+    companion object {
+        val create: ClipboardSupportFactory = { context ->
```

Comment:
using `create` instead of `shared` to denote that this is created on-demand and not allocated for the lifecycle of the containing `ClipboardSupport` class

Reply:
why would you use this strategy instead of a convenience initializer that takes a `Context`?

Reply:
TBH, I was trying out a few things to get to the point of working, and didn't go back to revisit this.  Looking it over, I think just having the initializer take a `Context` handles all the cases.